core/config/config.js

```diff
@@ -192,66 +191,34 @@ function overrideSettingsForGatherMode(settings, gatherMode) {
 /**
  * Overrides the quiet windows when throttlingMethod requires observation.
  *
- * @param {LH.Config.NavigationDefn} navigation
  * @param {LH.Config.Settings} settings
  */
-function overrideNavigationThrottlingWindows(navigation, settings) {
-  if (navigation.disableThrottling) return;
+function overrideThrottlingWindows(settings) {
   if (settings.throttlingMethod === 'simulate') return;
 
-  navigation.cpuQuietThresholdMs = Math.max(
-    navigation.cpuQuietThresholdMs || 0,
-    nonSimulatedPassConfigOverrides.cpuQuietThresholdMs
+  settings.cpuQuietThresholdMs = Math.max(
+    settings.cpuQuietThresholdMs || 0,
+    nonSimulatedSettingsOverrides.cpuQuietThresholdMs
   );
-  navigation.networkQuietThresholdMs = Math.max(
-    navigation.networkQuietThresholdMs || 0,
-    nonSimulatedPassConfigOverrides.networkQuietThresholdMs
+  settings.networkQuietThresholdMs = Math.max(
+    settings.networkQuietThresholdMs || 0,
+    nonSimulatedSettingsOverrides.networkQuietThresholdMs
   );
-  navigation.pauseAfterFcpMs = Math.max(
-    navigation.pauseAfterFcpMs || 0,
-    nonSimulatedPassConfigOverrides.pauseAfterFcpMs
+  settings.pauseAfterFcpMs = Math.max(
+    settings.pauseAfterFcpMs || 0,
+    nonSimulatedSettingsOverrides.pauseAfterFcpMs
   );
-  navigation.pauseAfterLoadMs = Math.max(
-    navigation.pauseAfterLoadMs || 0,
-    nonSimulatedPassConfigOverrides.pauseAfterLoadMs
+  settings.pauseAfterLoadMs = Math.max(
+    settings.pauseAfterLoadMs || 0,
+    nonSimulatedSettingsOverrides.pauseAfterLoadMs
   );
 }
 
-/**
- * @param {LH.Config.AnyArtifactDefn[]|null|undefined} artifactDefns
- * @param {LH.Config.Settings} settings
- * @return {LH.Config.NavigationDefn[] | null}
- */
-function resolveFakeNavigations(artifactDefns, settings) {
-  if (!artifactDefns) return null;
-
-  const status = {msg: 'Resolve navigation definitions', id: 'lh:config:resolveNavigationsToDefns'};
-  log.time(status, 'verbose');
-
-  const resolvedNavigation = {
-    ...defaultNavigationConfig,
-    artifacts: artifactDefns,
-    pauseAfterFcpMs: settings.pauseAfterFcpMs,
-    pauseAfterLoadMs: settings.pauseAfterLoadMs,
-    networkQuietThresholdMs: settings.networkQuietThresholdMs,
-    cpuQuietThresholdMs: settings.cpuQuietThresholdMs,
-    blankPage: settings.blankPage,
-  };
-
-  overrideNavigationThrottlingWindows(resolvedNavigation, settings);
-
-  const navigations = [resolvedNavigation];
-  assertArtifactTopologicalOrder(navigations);
-
-  log.timeEnd(status);
-  return navigations;
-}
-
 /**
  * @param {LH.Gatherer.GatherMode} gatherMode
  * @param {LH.Config=} config
  * @param {LH.Flags=} flags
- * @return {Promise<{resolvedConfig: LH.Config.ResolvedConfig, warnings: string[]}>}
```

Comment:
There are no config warnings anymore but there could be in the future I guess. Shouldn't be too hard to add this if we ever want them again.